docs/en/sql-reference/aggregate-functions/reference/argmin.md

```diff
@@ -49,3 +50,65 @@ Result:
 │ worker               │
 └──────────────────────┘
 ```
+
+**Extended example**
+
+```sql
+CREATE TABLE test
+(
+    a Nullable(String),
+    b Nullable(Int64)
+)
+ENGINE = Memory AS
+SELECT *
+FROM values((NULL, 0), ('a', 1), ('b', 2), ('c', 2), (NULL, NULL), ('d', NULL));
+
+select * from test;
+┌─a────┬────b─┐
+│ ᴺᵁᴸᴸ │    0 │
+│ a    │    1 │
+│ b    │    2 │
+│ c    │    2 │
+│ ᴺᵁᴸᴸ │ ᴺᵁᴸᴸ │
+│ d    │ ᴺᵁᴸᴸ │
+└──────┴──────┘
+
+select argMin(a, b), min(b) from test;
+┌─argMin(a, b)─┬─min(b)─┐
+│ a            │      0 │ -- argMin = a because it the first not-Null value, min(b) is from another row!
+└──────────────┴────────┘
+
+select argMin(tuple(a), b) from test;
+┌─argMin(tuple(a), b)─┐
+│ (NULL)              │ -- Tuple allows to get Null value.
```

Comment:
```suggestion
│ (NULL)              │ -- The a `Tuple` that contains only a `NULL` value is not `NULL`, so the aggregate functions won't skip that row because of that `NULL` value
```